src/AddTodoForm.jsx

```diff
@@ -22,14 +23,15 @@ function AddTodoForm({ onAddTodo }) {
     };
     return (
         <form onSubmit={handleAddTodo}>
-            <label htmlFor="todoTitle">Title </label>
-            <input
+            <InputWithLabel
                 id="todoTitle"
                 type="text"
                 name="title"
```

Comment:
seems you have a small issue with prop naming 
<img width="572" alt="image" src="https://github.com/user-attachments/assets/022eb671-b8f2-46d7-9a8d-e1e14f27b541">